aspnetcore/security/authentication/social/facebook-logins.md

```diff
@@ -62,26 +62,40 @@ Link sensitive settings like Facebook `App ID` and `App Secret` to your applicat
 
 ## Configure Facebook middleware
 
-> [!NOTE]
-> The project template used in this tutorial ensures that 
-[Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) package is already installed.
->
-> * To install this package with Visual Studio 2017, right-click on the project and select **Manage NuGet Packages**.
-> * To install with **dotnet** CLI, execute the following in your project directory:
->
->   `dotnet add package Microsoft.AspNetCore.Authentication.Facebook`
+The project template used in this tutorial ensures that [Microsoft.AspNetCore.Authentication.Facebook](https://www.nuget.org/packages/Microsoft.AspNetCore.Authentication.Facebook) package is already installed.
 
-Add the Facebook middleware in the `ConfigureServices` method in the *Startup.cs* file:
+* To install this package with Visual Studio 2017, right-click on the project and select **Manage NuGet Packages**.
+* To install with **dotnet** CLI, execute the following in your project directory:
+
+   `dotnet add package Microsoft.AspNetCore.Authentication.Facebook`
+
+# [ASP.NET Core 1.x](#tab/aspnet1x)
+
+Add the Facebook middleware in the `Configure` method in `Startup.cs` file:
+
+```csharp
+app.UseFacebookAuthentication(new FacebookOptions()
+{
+    AppId = Configuration["Authentication:Facebook:AppId"],
+    AppSecret = Configuration["Authentication:Facebook:AppSecret"]
+});
+```
+
+# [ASP.NET Core 2.0](#tab/aspnet20)
+
+Add the Facebook middleware in the `ConfigureServices` method in the `Startup.cs` file:
 
 ```csharp
-services.AddFacebookAuthentication(facebookOptions =>
+services.AddFacebookAuthentication(new FacebookOptions()
 {
-    facebookOptions.AppId = Configuration["Authentication:Facebook:AppId"];
-    facebookOptions.AppSecret = Configuration["Authentication:Facebook:AppSecret"];
+    AppId = Configuration["Authentication:Facebook:AppId"];
+    AppSecret = Configuration["Authentication:Facebook:AppSecret"];
 });
 ```
 
-Note: See the [FacebookOptions](https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.Facebook/FacebookOptions.cs) class in ASP.NET Core repository for more information on configuration options supported by Facebook middleware. Configuration options can be used to:
+---
+
+See the [FacebookOptions](https://github.com/aspnet/Security/blob/dev/src/Microsoft.AspNetCore.Authentication.Facebook/FacebookOptions.cs) class in ASP.NET Core repository for more information on configuration options supported by Facebook middleware. Configuration options can be used to:
```

Comment:
This makes no sense, I will leave this change to you if you are sure. The sentence was meant to establish context for a list of items, there is no "also" / "in addition to" meaning in it.